### ISSUE TITLE
Reproducablity basics?

### ISSUE THREAD

**Issue by aytackanaci** [zlai0/MAST#27]
Hi Zhiang,
I'm trying to reproduce your results howerwer I'm confused what to change in the training script for the 2 phases or training described in the paper.

Step 1:
_"During training, we first pretrain the network with a pair of
input frames, i.e. one reference frame and one target frame
are fed as inputs. One of the color channels is randomly
dropped with probability p = 0:5. We train our model endto-
end using a batch size of 24 for 1M iterations with the
Adam optimizer. The initial learning rate is set to 1e-3,
and halved after 0.4M, 0.6M and 0.8M iterations."_

Step 2:
_"We then
finetune the model using multiple reference frames (our full
memory-augmented model) with a small learning rate of 2e-
5 for another 1M iterations"_

Cold you provide commands similar commands in the `scripts/train` for this purpose?
For example, do you user2gpus and set bsize=12, or 1 GPU and bsize=24.
And what is needed to change to enable "full_memort_augmented_model" in `main.py` for step 2?

if anyone else figured this out before please let me know
thanks,
